### DISCUSSION TITLE
Cannot access Globals set variables from template

### DISCUSSION THREAD

**Discussion by giancarloerra** [statamic/cms#9368] [Troubleshooting] [answered]
Hi all,

I created from CP a new Global Set called Branding, and within its Blueprint an Asset Field pointing to my /images container in /public/images, which has URL '/images' in config/filesystems:

```
        'images' => [
            'driver' => 'local',
            'root' => public_path('images'),
            'url' => '/images',
            'visibility' => 'public',
        ],
```

This is the resulting branding.yml

```
title: Branding
data:
  site_logo:
    - header_logo_website_transparent.svg
```

In my template, if I point manually to the image it works:
`<img src="/images/header_logo_website_transparent.svg" />`

But if I try getting the Global variable it prints out nothing (as in src=""):
`<img src="{{ branding:site_logo }}" />`

I also trying clearing the stache, what am I missing?

**Comment by duncanmcclean** [comment, marked as answer]
Hey 👋

It looks like you're storing the `site_logo` as an array of assets, rather than just a single asset. Unless you have multiple logos, you probably want to set "Max Items" to 1 in your asset field's config so the syntax you're using works.

Otherwise, the Assets field would allow you to select *multiple* assets for the logo and you'd have to do this to make it work:

```antlers
{{ branding:site_logo }}
    <img src="{{ url }}" /> 
{{ /branding:site_logo }}
```

You can see more information about [Templating w/ the Assets Fieldtype here](https://statamic.dev/fieldtypes/assets#templating).
